proposals/lip-0051.md

```diff
@@ -1171,17 +1123,17 @@ transferCrossChain(timestamp,
         }       
         serializedParams = serialization of messageParams
                            following crossChainTransferMessageParams    
-        interoperability.send(timestamp,
+        interoperability.send(senderAddress,
```

Comment:
Similar to the NFT module, I think we also need to account for the `send` API failing. Probably it is then best first to send and afterwards to adjust account balances and escrow.

Reply:
Updated similarly to the NFT LIP.